ts/session/messages/outgoing/content/data/ChatMessage.ts

```diff
@@ -0,0 +1,156 @@
+import { DataMessage } from './DataMessage';
+import { SignalService } from '../../../../../protobuf';
+import { MessageParams } from '../../Message';
+import { LokiProfile } from '../../../../../types/Message';
+
+interface AttachmentPointer {
```

Comment:
Might be worth pulling this out into its own type. This is because we probably are going to use it in other message types.